std/algebra/emulated/fields_bls12381/e12_pairing.go

```diff
@@ -387,11 +387,12 @@ func (e Ext12) FrobeniusSquareTorus(y *E6) *E6 {
 // FinalExponentiationCheck checks that a Miller function output x lies in the
 // same equivalence class as the reduced pairing. This replaces the final
 // exponentiation step in-circuit.
-// The method follows Section 4 of [On Proving Pairings] paper by A. Novakovic and L. Eagen.
+// The method is inspired from [On Proving Pairings] paper by A. Novakovic and
+// L. Eagen, and is based on a personal communication with A. Novakovic.
 //
 // [On Proving Pairings]: https://eprint.iacr.org/2024/640.pdf
 func (e Ext12) FinalExponentiationCheck(x *E12) *E12 {
```

Comment:
We have denoted return type, but don't return anything. Remove returned value.

Reply:
done. also for other curves.